spec/arel/middleware_spec.rb

```diff
@@ -505,7 +505,8 @@ def self.call(_arel, next_middleware)
     expect(post.attributes).to eq('id' => nil, 'title' => 'some title')
   end
 
-  it 'does not use middleware when configuring a connection to prevent endless checkouts' do
+  it 'does not use middleware when configuring a connection to prevent endless checkouts',
+     database_cleaner: :truncation do
```

Comment:
Need to truncate here because the post is created in a different connection, so the transaction rollback does not work